src/Config.php

```diff
@@ -74,6 +74,9 @@ protected function configureOptions(OptionsResolver $resolver)
             'constructorParamsDefaultToNull' => false,
             'soapClientClass'               => '\SoapClient',
             'soapClientOptions'             => array(),
+            'soapServerClass'               => '\SoapServer',
+            'soapServerOptions'             => array(),
+            'serverClassName'               => '',
```

Comment:
Yes it's useful because the client class gets its name from the wsdl file (($service->getIdentifier()), but the server class name is not defined in the wsdl file, so I find it nice to let the user decide for the name ;)